contributor-day.md

```diff
@@ -12,6 +12,15 @@ Your leads for the day are: [schlessera](https://github.com/schlessera), [daniel
 
 A few seasoned WP-CLI contributors are also helping out: TBD
 
+## Open Video Sessions
+
+During the Hack Day, we’ll have two open video chat sessions:
+
+- From 🕓 08:00 WET to 09:00 WET
+- From 🕓 15:00 WET to 16:00 WET
```

Comment:
Let's show this in multiple timezones to make it easier for folks

```suggestion
- From 🕓 00:00-01:00 PST / 03:00-04:00 EST / 08:00-07:00 UTC /  09:00-10:00 CET / 17:00-18:00 JST
- From 🕓 07:00-08:00 PST / 10:00-11:00 EST / 15:00-16:00 UTC / 16:00-17:00 CET / 00:00-01:00 JST
```

I hope these are correct 😄